src/integration/utils/fetchDom.js

```diff
@@ -1,5 +1,12 @@
 /* eslint-disable no-console */
+
 const fetch = require('isomorphic-fetch');
+
+// https://github.com/node-fetch/node-fetch/issues/1624#issuecomment-1407717012
```

Comment:
Node 17 onwards defaults dns lookup to IPv6, so this sets it back to IPv4 for tests.

Think this should be ok declared here, may need moved higher up if we start seeing `ECONNREFUSED` errors in the CI/CD.